lib/package.json

```diff
@@ -53,14 +46,11 @@
         "@rollup/plugin-eslint": "^8.0.0",
         "@rollup/plugin-node-resolve": "^10.0.0",
         "@rollup/plugin-replace": "^2.3.4",
-        "@types/base64url": "^2.0.0",
-        "@types/crypto-js": "^3.1.43",
         "@types/node": "^13.9.2",
         "@types/randombytes": "^2.0.0",
```

Comment:
We can remove this as well, right?

Reply:
```json
"@types/randombytes": "^2.0.0"
```


This is required by `serialize-javascript` which is a dependency of rollup plugin

Reply:
removed